hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -583,7 +643,12 @@ private void notifyRemoteCall(Response response) {
     /**
      * Responsible for processing an Operation.
      */
-    private final class OperationHandler implements BasicOperationHandler {
+    private final class BasicOperationHandler implements OperationHandler {
```

Comment:
We'll rip out more stuff; I'm thinking about a package called basicoperationservice and than moving most of the inner classes as regular class. But not now.. else I won't get it merged.